src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -29,7 +29,7 @@
 #define UCT_RC_MLX5_AM_MAX_HDR(_qp_type) \
     UCT_IB_MLX5_MAX_BB * MLX5_SEND_WQE_BB - \
     (sizeof(struct mlx5_wqe_inl_data_seg) + \
-     sizeof(struct mlx5_wqe_data_seg) + \
+     UCT_IB_MAX_IOV * sizeof(struct mlx5_wqe_data_seg) + \
```

Comment:
1. I think we need another constant for this: UCT_IB_MLX5_AM_ZCOPY_MAX_IOV. because it should be quite small.
2. For DC, need to also reduce the address vector size. See UCT_RC_MLX5_AM_MAX_SHORT.
3. do we have a test which sends am_zcopy with maximal possible header size, and maximal possible iov size?